app/controllers/todos_controller.rb

```diff
@@ -1,70 +1,50 @@
 class TodosController < ApplicationController
-  before_action :set_todo, only: %i[ show edit update destroy ]
+  before_action :set_todo, only: [:show, :edit, :update, :destroy]
 
-  # GET /todos or /todos.json
   def index
     @todos = Todo.all
   end
 
-  # GET /todos/1 or /todos/1.json
   def show
   end
 
```

Comment:
The removal of JSON response handling in the create, update, and destroy actions may affect API clients expecting JSON responses. Consider maintaining JSON response handling for API compatibility.